drivers/periph_common/Kconfig.flashpage

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+menuconfig MOD_PERIPH_FLASHPAGE
+    bool "Flashpage peripheral driver"
+    depends on HAS_PERIPH_FLASHPAGE
+    select MOD_PERIPH_COMMON
+
+# TODO: the 'init' modules are actually just artifacts from the way
+# periph_init_% modules are handled in Makefile. We need to define them to keep
+# the list the same.
```

Comment:
so does this mean that these lines can be removed later on? If yes, then we should add that intent to the comments.

Reply:
AFAIK yes, they could be removed later on, as I could not find any references to them. I can adapt the comments.

Reply:
I have extended the comment to clarify this.